bait-files/src/test/java/com/google/sps/UsernameTakenTest.java

```diff
@@ -125,15 +119,16 @@ public void doGet_nothingInDb_returnsFalse() throws IOException, ServletExceptio
 
   @Test
   public void doGet_usernameIsNull_returnsTrue() throws IOException, ServletException {
+    //Returns true in first if statement in test
```

Comment:
haha... okay, but *why*? Why do we expect or want this behavior, in regards to the application?

Reply:
Refactored comment